figures/tasks.py

```diff
@@ -380,4 +380,7 @@ def run_figures_monthly_metrics():
     """
     logger.info('Starting figures.tasks.run_figures_monthly_metrics...')
     for site in get_sites():
-        populate_monthly_metrics_for_site.delay(site_id=site.id)
+        populate_monthly_metrics_for_site.apply_async(
```

Comment:
@melvinsoft Woudl you please add a comment as to why this code is explicitly setting the queue. Maybe also as a module level docstring for future Figures developers who will rewrite the daily tasks. Would be good to warn them that they need to be explicit in setting the queue for any `apply_async` calls so they don't go to any old queue.